File: src/main/java/br/com/mecanopecas/mecanopecas/util/dtos/response/OrcamentoDetailResponseDTO.java
package br.com.mecanopecas.mecanopecas.util.dtos.response;

import java.time.LocalDateTime;
import java.util.List;

public record OrcamentoDetailResponseDTO(Long id, String codigo, String clienteNome, String vendedorNome, LocalDateTime DataOrcamento, LocalDateTime dataExpiracao, Double valor, Double percentualDesconto, Double valorTotal, List<OrcamentoPecaResponseDTO> pecas, String status) {
}

File: front-mecanopecas/src/dtos/response/orcamentoDetailResponseDTO.ts
import { OrcamentoPecaResponseDTO } from './orcamentoPecaResponseDTO';

export interface OrcamentoDetailResponseDTO {
    id: bigint;
    codigo: string;
    clienteNome: string;
    vendedorNome: string;
    DataOrcamento: Date;
    dataExpiracao: Date;
    valor: number;
    percentualDesconto: number;
    valorTotal: number;
    pecas: OrcamentoPecaResponseDTO[];
    status: string;
}
